pom.xml

```diff
@@ -86,7 +86,7 @@
         <htmlunit.version>2.38.0</htmlunit.version>
         <activation.version>1.1.1</activation.version>
         <kotlin.version>1.3.71</kotlin.version>
-        <log4j-api.version>2.13.3</log4j-api.version>
+        <log4j-api.version>2.16.0</log4j-api.version>
```

Comment:
2.17; but will be fixed with the other PR, right?

Reply:
Right, since #165 removes log4j completely there is no need to bump this version for this release. There is a release version 1.4.2 that has the right version for Log4J.